app/helpers/analytics_helper.rb

```diff
@@ -0,0 +1,36 @@
+# -*- encoding : utf-8 -*-
+module AnalyticsHelper
+
+  # helpers for embedding Google Analytics code
+
+  def track_analytics_event(event_category, event_action, options={})
```

Comment:
As mentioned in https://github.com/mysociety/alaveteli/pull/3110#discussion_r56967008, it would be good to get some  [documentation](http://tomdoc.org/) about what the arguments do, particularly mentioning where to find the established categories and actions (i.e. `AnalyticsEvent::Category` and `AnalyticsEvent::Action`)